serialization/etf/src/test/java/net/automatalib/serialization/etf/writer/DFA2ETFWriterTest.java

```diff
@@ -45,4 +46,17 @@ public void testWrite() throws Exception {
 
         Assert.assertEquals(sw.toString().hashCode(), -1735686704);
     }
+
+    @Test
+    public void testEmptyLanguage() {
+        final StringWriter sw = new StringWriter();
+
+        final Alphabet<Character> alphabet = Alphabets.characters('a', 'a');
+        final DFA<?, Character> emptyLanguage =
+                AutomatonBuilders.newDFA(alphabet).withInitial("q0").from("q0").on('a').loop().create();
+
+        DFA2ETFWriter.write(sw, emptyLanguage, alphabet);
+
+        Assert.assertEquals(sw.toString().hashCode(), -204385940);
```

Comment:
Ideally we would want to have an ETF parser, but we only have an ETF writer.  If we would have an ETF parser we could check it for 100% correctness, but we don't and I feel checking the hashcode is the best we can do now.

Reply:
Sure, having both serializer and deserializer, we could simply check for a model M if M is equivalent to `deserialize(serialize(M))`.

But even for the serializer alone, having a textual representation of the expected ouput (e.g. an ETF file) is much more convenient. If at any point the serializer will change (e.g. adding support for state shapes) it will be much more easy to adjust (and understand) the expected output in an ETF file, rather than having to deal with its hashcode repesentation.

Reply:
All right I shall add the resources.